package.json

```diff
@@ -82,96 +82,93 @@
+    "cypress": "^5.1.0",
+    "directory-tree": "^2.2.4",
+    "directory-tree-webpack-plugin": "^1.0.2",
     "duplexer": "^0.1.1",
     "eslint": "^7.8.1",
-    "eslint-loader": "^2.0.0",
+    "eslint-loader": "^4.0.2",
     "eslint-plugin-cypress": "^2.10.3",
     "eslint-plugin-markdown": "^1.0.2",
-    "file-loader": "^1.1.11",
-    "fontgen-loader": "git://github.com/EugeneHlushko/fontgen-loader.git#0.1.3",
-    "front-matter": "^2.3.0",
-    "gh-pages": "^1.0.0",
-    "html-webpack-plugin": "^3.2.0",
-    "html-webpack-template": "^6.1.0",
-    "http-server": "^0.10.0",
-    "husky": "^1.0.0-rc.8",
+    "eslint-webpack-plugin": "^2.1.0",
+    "file-loader": "^6.1.0",
+    "fontgen-loader": "git://github.com/chenxsan/fontgen-loader.git#bugfix/fix-this-exec",
+    "front-matter": "^4.0.2",
+    "gh-pages": "^3.1.0",
+    "html-loader": "^1.3.0",
+    "html-webpack-plugin": "^4.4.1",
+    "http-server": "^0.12.3",
+    "husky": "^4.3.0",
     "hyperlink": "^4.4.0",
-    "jest": "^23.6.0",
-    "lint-staged": "^8.1.0",
-    "loader-utils": "^1.1.0",
+    "jest": "^26.4.2",
+    "lint-staged": "^10.3.0",
     "lodash": "^4.17.19",
-    "markdownlint": "^0.11.0",
-    "markdownlint-cli": "^0.13.0",
+    "markdownlint": "^0.20.4",
+    "markdownlint-cli": "^0.23.2",
     "markdownlint-rule-emphasis-style": "^1.0.0",
-    "mini-css-extract-plugin": "^0.5.0",
+    "mini-css-extract-plugin": "^0.11.1",
     "minimist": "1.2.3",
-    "mkdirp": "^0.5.1",
+    "mkdirp": "^1.0.4",
     "modularscale-sass": "^3.0.3",
     "node-sass": "^4.5.3",
     "npm-run-all": "^4.1.1",
-    "offline-plugin": "^5.0.7",
-    "optimize-css-assets-webpack-plugin": "^5.0.1",
-    "postcss-loader": "^2.1.3",
-    "redirect-webpack-plugin": "^0.1.1",
-    "remark": "^10.0.1",
-    "remark-autolink-headings": "5.2.1",
+    "postcss": "^7.0.32",
+    "postcss-loader": "^4.0.1",
+    "redirect-webpack-plugin": "^1.0.0",
+    "remark": "^12.0.1",
+    "remark-autolink-headings": "^6.0.1",
     "remark-custom-blockquotes": "1.0.0",
     "remark-extract-anchors": "1.1.1",
-    "remark-loader": "^0.3.0",
+    "remark-html": "^12.0.0",
+    "remark-loader": "^1.0.0",
     "remark-refractor": "git://github.com/montogeek/remark-refractor.git",
     "remark-responsive-tables": "1.0.0",
-    "remark-slug": "^5.0.0",
+    "remark-slug": "^6.0.0",
     "request-promise": "4.2.4",
-    "rimraf": "^2.6.2",
-    "sass-loader": "^6.0.6",
-    "sirv-cli": "^0.1.2",
+    "rimraf": "^3.0.2",
+    "sass-loader": "^10.0.2",
+    "sirv-cli": "^1.0.6",
     "sitemap-static": "^0.4.2",
     "start-server-and-test": "^1.11.0",
     "static-site-generator-webpack-plugin": "^3.4.1",
-    "style-loader": "^0.18.2",
+    "style-loader": "^1.2.1",
     "tap-spot": "^1.1.1",
-    "terser-webpack-plugin": "^2.3.5",
-    "through2": "^2.0.3",
-    "uglifyjs-webpack-plugin": "^1.1.6",
-    "webpack": "^4.42.0",
+    "terser-webpack-plugin": "^4.1.0",
+    "webpack": "^5.0.0-beta.29",
```

Comment:
Let's update to `^5.0.0.-beta.30` :smile:

Reply:
It's already beta.30 https://github.com/webpack/webpack.js.org/pull/3963/commits/0b3920b742aae10b6c1f663f9a00d66411b7d817#diff-8ee2343978836a779dc9f8d6b794c3b2R14080 in yarn.lock

Reply:
Yep